shared/teams/new-team/wizard/create-subteams.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Container from '../../../util/container'
+import * as Styles from '../../../styles'
+import {pluralize} from '../../../util/string'
+import {ModalTitle} from '../../common'
+
+const cleanSubteamName = (name: string) => name.replace(/[^0-9a-zA-Z_]/, '')
+
+type Props = {
+  teamname: string
+}
+
+const CreateSubteams = (props: Props) => {
+  const dispatch = Container.useDispatch()
+  const nav = Container.useSafeNavigation()
+
+  const [subteams, setSubteams] = React.useState<Array<string>>(['', '', ''])
+  const setSubteam = (i: number, value: string) => {
+    subteams[i] = value
+    setSubteams([...subteams])
+  }
+  const onClear = (i: number) => {
+    subteams.splice(i, 1)
+    setSubteams([...subteams])
+  }
+  const onAdd = () => {
+    subteams.push('')
+    setSubteams([...subteams])
+  }
+
+  const onBack = () => dispatch(nav.safeNavigateUpPayload())
+
+  const numSubteams = subteams.filter(c => !!c.trim()).length
+  const continueLabel = numSubteams
+    ? `Continue with ${numSubteams} ${pluralize('subteam', numSubteams)}`
+    : 'Continue without subteams'
+
+  return (
+    <Kb.Modal
+      onClose={onBack}
+      header={{
+        leftButton: <Kb.Icon type="iconfont-arrow-left" onClick={onBack} />,
+        title: <ModalTitle teamname={props.teamname} title="Create subteams" />,
+      }}
+      footer={{content: <Kb.Button fullWidth={true} label={continueLabel} />}}
+      allowOverflow={true}
+    >
+      <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.banner} centerChildren={true}>
+        <Kb.Icon type="icon-illustration-teams-subteams-460-96" />
+      </Kb.Box2>
+      <Kb.Box2
+        direction="vertical"
+        fullWidth={true}
+        style={styles.body}
+        gap={Styles.isMobile ? 'xsmall' : 'tiny'}
+      >
+        <Kb.Text type="BodySmall">Channels can be joined by anyone in the team, unlike subteams.</Kb.Text>
+        {subteams.map((value, idx) => (
+          <Kb.NewInput
+            value={value}
+            onChangeText={text => setSubteam(idx, cleanSubteamName(text))}
+            decoration={<Kb.Icon type="iconfont-remove" onClick={() => onClear(idx)} />}
+            placeholder="subteam"
+            prefix={`${props.teamname}.`}
+            containerStyle={styles.input}
+            maxLength={16}
+            key={idx}
+          />
+        ))}
+        <Kb.Button
+          mode="Secondary"
+          icon="iconfont-new"
+          tooltip="Add another subteam"
```

Comment:
Hovering anywhere over the row with the button shows this
<img width="385" alt="image" src="https://user-images.githubusercontent.com/11968340/74972787-5ca82000-53f0-11ea-9df5-863b756548f3.png">